api.planx.uk/webhooks/sendNotifications.ts

```diff
@@ -24,11 +24,8 @@ const sendSlackNotification = (req: Request, res: Response, next: NextFunction):
       }
 
       const bopsMessage = `:incoming_envelope: New BOPS submission *${data?.bops_id}* [${data?.destination_url}]`;
-      slack.send(bopsMessage)
-        .then(() => {
-          res.status(200).send({ message: "Posted to Slack", data: bopsMessage });
-        })
-        .catch(error => next(error));
```

Comment:
This error wasn't getting passed to the `catch` block of the `try/catch` but getting passed directly to `errorHandler()` via `next()`. This meant that the error was losing some of it's context.

The small refactor here reads more clearly (to me), and now throws it's error to the `try/catch` handler wrapping this function.